infra/src/main/resources/db/migration/prod/V39__add_element_listing_file_table.sql

```diff
@@ -0,0 +1,8 @@
+create table layout.element_listing_file
+(
+  id          int primary key generated always as (1) stored,
+  name        varchar(100) not null,
+  content     varchar      not null,
+  change_time timestamptz  not null,
```

Comment:
Nämä metatiedot on pakko päivittää dao:sta käsin koska tälle ei haluta versiointia. Tein uuden tiketin siitä että eritellään muun metadatan päivitystriggereiden luontifunkkarit versioinnista jotta niitä voi käyttää myös itsenäisesti (näitä paikkoja on jo nyt ainakin 3): https://extranet.vayla.fi/jira/browse/GVT-1912